superset/views.py

```diff
@@ -1104,6 +1104,34 @@ def ping():
     return "OK"
 
 
+class KV(BaseSupersetView):
+
+    """used for storing and retrieving key value pairs"""
+
+    @log_this
+    @expose("/store/", methods=['POST', 'GET'])
+    def store(self):
+        value = request.form.get('data')
+        base_url = request.form.get('baseUrl') \
+            if request.form.get('baseUrl') else ''
+        obj = models.KeyValue(value=value)
+        db.session.add(obj)
+        db.session.commit()
+        return("http://{request.headers[Host]}/{base_url}?id={obj.id}".format(
```

Comment:
Avoiding redirects is why we opened this PR in the first place, because we want to avoid including queries in url params. We return the url in the same pattern in shortener